data/pages/data-privacy-de.md

```diff
@@ -9,8 +9,8 @@ language: de
    Daten sind alle Daten, die auf Sie persönlich beziehbar sind, z. B. Name, Adresse, E-Mail-Adressen, Nutzerverhalten.
 2. Verantwortlicher gem. Art. 4 Abs. 7 EU-Datenschutz-Grundverordnung (DSGVO) ist Satellytes Digital Consulting GmbH,
    Sendlinger Straße 52, 80331 München, info@satellytes.com (siehe [Impressum](/imprint/)).
-3. Bei Ihrer Kontaktaufnahme mit uns per E-Mail oder über ein Kontaktformular werden die von Ihnen mitgeteilten Daten (
-   Ihre E-Mail-Adresse, ggf. Ihr Name und Ihre Telefonnummer) von uns gespeichert, um Ihre Fragen zu beantworten. Die in
+3. Bei Ihrer Kontaktaufnahme mit uns per E-Mail oder über ein Kontaktformular werden die von Ihnen mitgeteilten Daten 
+   (Ihre E-Mail-Adresse, Ihr Vorname, ggf. Ihr Nachname und ggf. Ihr Lebenslauf) von uns gespeichert, um Ihre Fragen zu beantworten. Die in
```

Comment:
@georgiee It would be great if you could check if this is ok (also the English Version)